openassessment/xblock/ui_mixins/mfe/test_page_context_serializer.py

```diff
@@ -606,6 +619,29 @@ def test_self_assessment_not_available(self, xblock):
 
         self.assertNestedDictEquals(expected_data, progress_data)
 
+    @scenario('data/grade_scenario_self_staff.xml', user_id='Alan')
+    def test_self_staff_assessment__waiting(self, xblock):
+        # Given I am on the staff step and waiting for submissions
+        self.create_submission_and_assessments(xblock, 'submission_text', [], [], SELF_ASSESSMENT)
+        xblock.workflow_data.workflow['status'] = 'waiting'
```

Comment:
I don't think this line is doing anything. Did you mean for this to be
```suggestion
        assert xblock.workflow_data.workflow['status'] == 'waiting'
```